src/hotspot/share/gc/g1/g1ConcurrentMark.cpp

```diff
@@ -453,6 +453,14 @@ G1ConcurrentMark::G1ConcurrentMark(G1CollectedHeap* g1h,
   }
 
   reset_at_marking_complete();
+
+  if (UsePerfData && os::is_thread_cpu_time_supported()) {
+    EXCEPTION_MARK;
+
+    _g1_concurrent_mark_threads_cpu_time =
+        PerfDataManager::create_variable(NULL_NS, "g1_conc_mark_thread_time",
```

Comment:
See my general comment about name spaces. The name should be something like `*.cpu_time`.

Reply:
+1. We probably should avoid using `NULL_NS` as well. So perhaps something like:
```
PerfDataManager::create_variable(SUN_THREADS, "g1_conc_mark_cpu_time", ...
```
There is only one existing hsperf counter under `SUN_THREADS`: sun.threads.vmOperationTime.
I think it is appropriate to add all these thread CPU time counters under `SUN_THREADS`.